src/plugins/amcl/sensors/amcl_laser.cpp

```diff
@@ -56,6 +56,15 @@ AMCLLaser::AMCLLaser(size_t max_beams, map_t* map) : AMCLSensor()
   this->max_beams = max_beams;
   this->map = map;
 
+  this->model_type = LASER_MODEL_BEAM;
+  this->z_hit = .95;
+  this->z_short = .05;
+  this->z_max = .05;
+  this->z_rand = .05;
+  this->sigma_hit = .2;
+  this->lambda_short = .1;
+  this->chi_outlier = 0.0;
+
```

Comment:
Was this marked as resolved on accident?

Reply:
No, sometimes Github doesn't take the comment when you resolve it seems. Here it is again.

In this case, an initializer list has no benefit. These are primitive types, which are not initialized on construction. Therefore, there is no performance difference between an initializer list and an assignment in the ctor. I chose to keep the existing style and use assignments instead of an initializer list.